backend/controllers/posts-controller.ts

```diff
@@ -49,41 +49,41 @@ export const createPostHandler = async (req, res) => {
 
     const [savedPost] = await Promise.all([
       post.save(), // Save the post
-      deleteDataFromCache(REDIS_KEYS.ALL_POSTS), // Invalidate cache for all posts
-      deleteDataFromCache(REDIS_KEYS.FEATURED_POSTS), // Invalidate cache for featured posts
-      deleteDataFromCache(REDIS_KEYS.LATEST_POSTS), // Invalidate cache for latest posts
+      // deleteDataFromCache(REDIS_KEYS.ALL_POSTS), // Invalidate cache for all posts
```

Comment:
What is the problem basically we are facing here?

Reply:
[ErrorReply: NOAUTH Authentication required.] this is the error i got

Reply:
Can you please figure out why we require this, 
Have you setup redis locally in the said port given in the env? 

Once run redis locally, and see if you can figure out something from it, you would mostly like be able to do it!

Reply:
@krishnaacharyaa yes sir it is working well I just gave the correct port number.

Reply:
@krishnaacharyaa it shows the error in the backend/dist/server.js file (Object.defineProperty(exports, "__esModule", { value: true });) exports is not defined in the es module scope.

Reply:
This problem is resolved @krishnaacharyaa